routes/stubs/probationAreas.js

```diff
@@ -0,0 +1,79 @@
+module.exports = [
+  {
+    area: {
+      content: { code: 'Lon', description: 'London' },
```

Comment:
this could be simplified a bit: you don't need to represent `content` blocks.
You can optimise it to suit the most simple way of representing the data:

```js
{ 
 code:  "Lon",
 description:  "London",
 ldus: [
        { code: 'ham', description: 'Hampstead' },
        { code: 'wtl', description: 'Waterloo' },
        { code: 'pic', description: 'Picadilly' },
        { code: 'ham', description: 'Hampstead0' },
        { code: 'wtl', description: 'Waterloo' },
        { code: 'pic', description: 'Picadilly' },
        { code: 'ham', description: 'Hampstead1' },
        { code: 'wtl', description: 'Waterloo1' },
        { code: 'pic', description: 'Picadilly1' },
        { code: 'ham', description: 'Hampstead' },
        { code: 'wtl', description: 'Waterloo' },
        { code: 'pic', description: 'Picadilly' },
        { code: 'ham', description: 'Hampstead1' },
        { code: 'wtl', description: 'Waterloo2' },
        { code: 'pic', description: 'Picadilly1' },
      ],
}
```

Reply:
I did wonder about that but I tried to keep it in line with the image in the ticket

Reply:
the returned response needs to be in that format - but the structure we store that data can be optimised for that